skt/executable.py

```diff
@@ -420,7 +420,8 @@ def setup_parser():
 
     parser_build = subparsers.add_parser("build", add_help=False)
     parser_build.add_argument("-c", "--baseconfig", type=str,
-                              help="Path to kernel config to use")
+                              help="Path to kernel config to use",
+                              default=os.environ.get("BASECONFIG"))
```

Comment:
Same here, could you use name "SKT_BASECONFIG" instead?

Reply:
Done.